web/src/presence.ts

```diff
@@ -31,9 +31,11 @@ export type PresenceInfoFromEvent = {
 
 // In future commits we'll use raw_info to facilitate
 // handling server events and/or timeout events.
-const raw_info = new Map<number, RawPresence>();
+export const raw_info = new Map<number, RawPresence>();
 export const presence_info = new Map<number, PresenceStatus>();
 
+export const raw_info_for_unknown_users = new Map<number, RawPresence>();
```

Comment:
Why do we need a special store? I would have thought that we could just store the information in the main `presence_info` / `raw_info` stores, and just have display code only iterate through users that it knows above via the main `people.ts` structures -- I thought that's roughly what we did in `main` already...

Reply:
Yeah, this approach works too after all, we can add the unknown user to the data, just making the buddy list doesn't trip up on it and that we redraw the user upon receiving the user creation event if the presence data already raced before it.